app/assets/javascripts/components/editor.js

```diff
@@ -38,9 +38,21 @@ var Editor = (function($, autosize) {
          * @type {function[]}
          */
         this.commands = {
-            'text': function($context) {
-                return $context.data().editorMarkupText;
+            text: function($context) {
```

Comment:
The `$context` variable is the element that triggered the text insertion command (see [this line](https://github.com/roschaefer/story.board/blob/f94e157d58260064ebfc587bdcb8e633da24257f/app/assets/javascripts/components/editor.js#L82)) so that we can access e. g. the data attributes.